templates/web-pdb.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if $.Values.web.enabled }}
+{{- if and (gt ($.Values.web.replicaCount | int) 1) ($.Values.web.podDisruptionBudget) }}
```

Comment:
I would not filter based on replica count. There is a good chance someone might scale outside of helm later.

Reply:
This matches the other pdb template
https://github.com/temporalio/helm-charts/blob/master/charts/temporal/templates/server-pdb.yaml#L4

Reply:
I'll fix that one in a later PR.

Reply:
removed replicaCount check